float32[] jointAngle
float32 acc
float32 vel
float32 cycle
float32 lookaheadTime
float32 gain
